aptos-move/parallel-executor/src/proptest_types/bencher.rs

```diff
@@ -103,9 +103,14 @@ where
     }
 
     pub(crate) fn run(self) {
+        let thread_pool = rayon::ThreadPoolBuilder::new()
```

Comment:
I don't think we ever looked into fake bencher numbers too much, but still we probably shouldn't be creating the threadpool in run here. For other tests it's probably fine.